### DISCUSSION TITLE
On Linux it is more beautiful

### DISCUSSION THREAD

**Discussion by PatriaMinas** [pawelsalawa/sqlitestudio#4606] [General]
Just a curious fact I've noticed. It's no problem. The design of SqliteStudio on Linux Fedora is much prettier than on MS Windows. It has some benefit for Linux. I don't remember if version 3.3 had this difference.

No problem.

**Comment by pawelsalawa** [comment]
Perhaps it's a matter of a SQLiteStudio's UI style that you use under Linux and Windows?
